samples/csharp_dotnetcore/02.echo-bot/Startup.cs

```diff
@@ -25,7 +25,7 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            services.AddControllers().AddNewtonsoftJson();
```

Comment:
if we are being strict - this AddNewtonsoftJson() is not necessary here - is that correct?
it is only necessary if you were using skills and not using ExpectReplies